src/styles/colors.ts

```diff
@@ -33,10 +32,8 @@ export const forceTheme = (theme: ThemeChoices | null): void => {
  *
  * @param themeDefinition - Partial or full set of theme tokens to override.
  */
-export const overrideLightTheme = (themeDefinition: ThemeDefinition): void => {
-  if (themeDefinition && typeof themeDefinition === 'object') {
-    lightThemeOverride = themeDefinition;
-  }
+export const overrideLightTheme = (themeDefinition: ThemeOverride): void => {
+  lightThemeOverride = themeDefinition;
```

Comment:
This one was ok-ish, but the checks are needless since the type does not allow anything else